### DISCUSSION TITLE
Authenticated scanning with Nuclei

### DISCUSSION THREAD

**Discussion by mikemicky4321** [projectdiscovery/nuclei#4991] [General]
Hello Team,
We would like to implement authenticated based scan for an application like OWASP Juice shop. But we are receiving below error. Please find attached yaml files both login.yaml and secrets.yaml. I am getting below error. Could you please help us?

![image](https://github.com/projectdiscovery/nuclei/assets/115986628/bb1eb632-f32b-4808-8e5f-ebf0f5612a22)

[yamlfiles.zip](https://github.com/projectdiscovery/nuclei/files/14872886/yamlfiles.zip)

![image](https://github.com/projectdiscovery/nuclei/assets/115986628/0a3aec6f-a668-447a-b946-46e1679e1fa8)

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
